golden-ratio.el

```diff
@@ -152,10 +152,12 @@ will prevent the window to be resized to the golden ratio."
           (golden-ratio-mode nil))
       ;; Always disable `golden-ratio-mode' to avoid
       ;; infinite loop in `balance-windows'.
-      (balance-windows)
-      (golden-ratio--resize-window dims)
-      (when golden-ratio-recenter
-        (scroll-right) (recenter)))))
+      (let (window-configuration-change-hook)
+        (balance-windows)
+        (golden-ratio--resize-window dims)
+        (when golden-ratio-recenter
+          (scroll-right) (recenter)))
+      (run-hooks 'window-configuration-change-hook))))
```

Comment:
No, the `let` is not malformed.

> I don't think this does what you think it does.

Can you prove it? You can check by runing this code:
```elisp
(defvar golden-ration-window-config-hook-count 0)
(add-hook 'window-configuration-change-hook
                 #'(lambda () 
                     (incf golden-ration-window-config-hook-count)
                     (message "wcch-count: %s" golden-ration-window-config-hook-count))
```
And see the difference in speed with wich the counter grows without my patch and with it.